scripts/verify

```diff
@@ -90,6 +90,18 @@ kubectl create namespace "$NAMESPACE"
 if [[ "$istio" = "enabled" ]]; then
   echo "INFO Enabling istio in \"$NAMESPACE\" (istio-injection=enabled)."
   kubectl label namespace "$NAMESPACE" istio-injection=enabled
+
+  istio_enabled_key="$(extract_schema_key.py \
```

Comment:
it's possible that there are multiple ISTIO_ENABLED keys. Use `head -n 1` to make sure we only get the first key

Can we add a TODO and open an issue to track handling of multiple keys?